data/json/professions.json

```diff
@@ -3709,5 +3709,35 @@
       },
       "female": [ "chestwrap" ]
     }
+  },
+  {
+  "type": "profession",
+  "ident": "politician",
+  "name": "Career Politician",
+  "description": "You've spent your life appealing to the people, persuading many and promising much throughout your time in office.  Now that your voting base wants to eat you alive, winning hearts and minds just got that much harder.",
+  "points": 4,
+  "skills": [
+    { "level": 4, "name": "barter" },
+    { "level": 6, "name": "speech" }
+  ],
+  "traits": [
+    "LIAR"
+  ],
+  "items": {
+    "both": [
+      "suit",
+      "tieclip",
+      "sock_wool",
+      "dress_shoes",
+      "smart_phone",
+      "gold_watch",
+      "briefcase",
+      "skinny_tie",
+      "file",
+      "ref_lighter",
+      "cigar"
+    ],
+    "male": [ "boxer_shorts" ],
+    "female": [ "bra", "panties" ]
   }
-]
+}
```

Comment:
Don't remove the empty line after this when you lint, that's why it shows the red error sign.

Reply:
So put in one empty line—3744, which will be blank—and that will fix the issue?

Reply:
Yes, that will fix it. It threw me for a loop on my first PR too

Reply:
Wrong bracket at the end, needs to be square( ] ) rather than curly ( } ) for the end of a .json

Reply:
Alright. I _think_ that this should solve everything. I sure hope it does, anyways.